packages/core/src/routes/admin-user/basics.ts

```diff
@@ -54,20 +54,16 @@ export default function adminUserBasicsRoutes<T extends ManagementApiRouter>(
     async (ctx, next) => {
       const {
         params: { userId },
-        query: { includeSsoIdentities },
+        query: { includeSsoIdentities = 'false' },
       } = ctx.guard;
 
       const user = await findUserById(userId);
 
-      ctx.body = {
-        ...pick(user, ...userInfoSelectFields),
-        ...conditional(
-          includeSsoIdentities &&
-            yes(includeSsoIdentities) && {
-              ssoIdentities: await findUserSsoIdentities(userId),
-            }
+      ctx.body = transpileUserProfileResponse(user, {
```

Comment:
it'll be great if we can implement some methods to

- check if sensitive fields are stripped; or
- automatically stripe sensitive fields

can be a global middleware or something, or we can just start with adding these checks in our test cases. not a blocking issue.